homeassistant/components/whirlpool/config_flow.py

```diff
@@ -0,0 +1,73 @@
+"""Config flow for Whirlpool Sixth Sense integration."""
+import logging
+
+import aiohttp
+import voluptuous as vol
+from whirlpool.auth import Auth
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({CONF_USERNAME: str, CONF_PASSWORD: str})
+
+
+async def validate_input(hass: core.HomeAssistant, data):
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+
+    auth = Auth(data[CONF_USERNAME], data[CONF_PASSWORD])
+    try:
+        await auth.do_auth()
+    except aiohttp.ClientConnectionError as exc:
```

Comment:
What about other aiohttp client errors

Reply:
Looks like this comment is still outstanding